### ISSUE TITLE
Format message arguments

### ISSUE THREAD

**Issue by edoshor** [exoscale/python-logstash-formatter#16]
We get messages with %s in them and an *args* field. Should it format the message with the provided arguments ?

logging.Formatter does
```python
s = self._fmt % record.__dict__
```

**Comment by andrewmichaelsmith** [comment]
+1 This was the behaviour I expected

**Comment by pyr** [comment]
agreed.

**Comment by pyr** [comment]
The actual behavior is to have the message do

```python
message.format(fields)
```

Which means references to fields will have to be done as for format:

```python
logger.warning("account {account} is in status {status}", extra={'account': 'foo', 'status': 'bar'})
```

**Comment by pyr** [comment]
(released as 0.5.13)
